tree/dataframe/inc/ROOT/RDFActionHelpers.hxx

```diff
@@ -349,12 +349,22 @@ public:
 
    void Initialize() { /* noop */}
 
-   void Finalize() { fTo->Merge(); }
+   void Finalize()
+   {
+      auto resObj = fObjects[0];
+      const auto nSlots = fObjects.size();
+      TList l;
+      l.SetOwner();
```

Comment:
this is a pretty sneaky way to `delete` elements 1 to N of `fObjects` -- I think it deserves at least a comment for people ctrl-F'ing `delete` after seeing the manual `new`.

Reply:
Yes, ROOT ownership is sneaky.